### PR TITLE
More api

### PR DESCRIPTION
**Pull request by turuslan**
### Description of the Change
More api.

### Benefits
More api.

### Possible Drawbacks 
None.

### CODE REVIEW

File: core/api/make.cpp [hunk 1]
```diff
@@ -174,13 +283,19 @@ namespace fc::api {
...
+        .StateGetActor = {[=](auto &address,
                               auto &tipset_key) -> outcome::result<Actor> {
           OUTCOME_TRY(context, tipsetContext(tipset_key, true));
           return context.state_tree.get(address);
         }},
+        .StateListMiners = {[=](auto &tipset_key)
+                                -> outcome::result<std::vector<Address>> {
+          OUTCOME_TRY(context, tipsetContext(tipset_key));
+          OUTCOME_TRY(power_state, context.powerState());
+          return power_state.claims.keys();
```
Comment (Alexey-N-Chernyshov) [inline core/api/make.cpp:295]:
```suggestion
          return power_state.getMiners();
```

Comment (Alexey-N-Chernyshov) [reply core/api/make.cpp:295]: `getMiners()` filters out miners with 0 pledge while `claims.keys()` doesn't.

Comment (turuslan) [reply core/api/make.cpp:295]: No, see [the-part#yeNsEL](https://the-part.t-b-m.space/doc/yeNsEL)

Comment (Alexey-N-Chernyshov) [reply core/api/make.cpp:295]: ok

File: core/api/make.cpp [hunk 2]
```diff
@@ -64,11 +68,11 @@ namespace fc::api {
 
   Api makeImpl(std::shared_ptr<ChainStore> chain_store,
                std::shared_ptr<WeightCalculator> weight_calculator,
-               std::shared_ptr<IpfsDatastore> ipld,
+               std::shared_ptr<Ipld> ipld,
                std::shared_ptr<BlsProvider> bls_provider,
                std::shared_ptr<KeyStore> key_store) {
     auto chain_randomness = chain_store->createRandomnessProvider();
-    auto tipsetContext = [&](auto &tipset_key,
+    auto tipsetContext = [=](auto &tipset_key,
```
Comment (artyom-yurin) [inline core/api/make.cpp:75]: Why by value?

**Pull request merged**